core/mosaic-dev-utils/get-folder-size.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-restricted-syntax */
+const fs = require('fs');
+const path = require('path');
+
+const getFolderSizeFn = async (dirPath, size = 0) => {
+    let result = 0;
+    const fFiles = [];
+
+    for (const file of await fs.promises.readdir(dirPath, { withFileTypes: true, encoding: 'utf-8' })) {
```

Comment:
There should be no awaits in for loop. Please refactpr to remove it. The promise all is here to help eliminate the request in the loop performance problem :)

Reply:
Hello! I refactored code, but kind not sure if I'm doing it right. Tbh, first time working with promise.all and files in node.js at all.

Reply:
This is not a performance problem, just a different approach to asynchronous operations.
Although, this is indeed can be refactored to use Promise.all approach, since we don't depend on results from the previous cycle.